README.md
Altitude responsive Wordpress theme
====================================

- Altitude
- by CyberChimps http://cyberchimps.com
- Licensed under GNU General Public License v3.0 - http://www.gnu.org/licenses/gpl-3.0.html

- Bootstrap
- used for grid structure
- by Twitter Bootstrap http://getbootstrap.com/2.3.2/
- Licensed under APLv2 - https://github.com/twitter/bootstrap/blob/master/LICENSE

- Genericons
- fonticons for arrows, visual links etc
- http://genericons.com/
- Licensed under GNU General Public License v2.0 - http://www.gnu.org/licenses/gpl-2.0.html

- Source Sans Pro
- Main font
- https://www.google.com/fonts#UsePlace:use/Collection:Source+Sans+Pro
- Licensed under SIL OPEN FONT LICENSE Version 1.1

- All other images and fonts
- by CyberChimps http://cyberchimps.com
- Licensed under GNU General Public License v3.0 - http://www.gnu.org/licenses/gpl-3.0.html

====================================================================================================================================

For updated documentation, walkthroughs, and support please visit http://cyberchimps.com/

For updated docs please visit http://cyberchimps.com/guides/

For the support forum please visit: http://cyberchimps.com/forum/

changelog.txt
Altitude

Version 1.0.2 (09/12/2014)

- Added WooCommerce support.
- WooCommerce template fix.
- New template with widgets.

Version 1.0.1 (07/25/2014)

- Added social icons to customizer

Version 1.0.0 (07/22/2014)

- Initial setup

front-page.php
<?php
/**
 * Home Template
 *
 * Please do not edit this file. This file is part of the CyberChimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */
?>

<?php if ( 'posts' == get_option('show_on_front') ) : ?>

	<?php get_header(); ?>

	<div id="primary" class="content-area container">

		<?php if ( get_theme_mod( 'altitude_home_title') != '' || get_theme_mod( 'altitude_home_sub_title') != '' ) : ?>
			<header class="home-header">
				<h1 class="home-title"><?php echo get_theme_mod( 'altitude_home_title', __( 'Altitude Features', 'altitude' ) ); ?></h1>
				<h2 class="home-sub-title"><?php echo get_theme_mod( 'altitude_home_sub_title', __( 'Packed full of great features, here are just a few', 'altitude' ) ); ?></h2>
				<hr class="s-divider" />
			</header>
		<?php endif; ?>

		<main id="main" class="site-main home-posts" role="main">

			<?php
			$args = array(
				'posts_per_page'      => 4,
				'ignore_sticky_posts' => 1
			);

			$query = new WP_Query( $args );
			?>

			<?php if ( $query->have_posts() ) : ?>

				<?php /* Start the Loop */ ?>
				<?php while ( $query->have_posts() ) : $query->the_post(); ?>

					<article id="post-<?php the_ID(); ?>" <?php post_class('s-col-3 isotope-item'); ?>>
						<div class="article-container">

							<div class="row">
								<div class="xs-col-12">

									<header class="entry-header">
										<h1 class="entry-title"><a href="<?php the_permalink(); ?>"><?php the_title(); ?></a></h1>
										<div class="entry-thumbnail"><?php the_post_thumbnail( 'altitude-home' ); ?></div>
									</header><!-- .entry-header -->

									<div class="entry-summary">
										<div class="entry-content"><?php the_content(''); ?></div>
									</div><!-- .entry-summary -->

									<footer class="entry-footer">
										<a class="read-more" href="<?php the_permalink(); ?>"><?php _e('Read More', 'altitude'); ?></a>
									</footer>

								</div>
							</div><!-- row -->

						</div><!-- article-container -->
					</article><!-- #post-## -->

				<?php endwhile; ?>

			<?php endif; ?>

			<?php wp_reset_postdata(); ?>

		</main><!-- #main -->

	</div><!-- #primary -->

	<?php get_template_part( 'home', 'widgets' ); ?>

	<?php get_footer(); ?>

<?php elseif ( 'page' == get_option('show_on_front') ) : ?>

	<?php include( get_page_template() ); ?>

<?php else : ?>

	<?php include( get_home_template() ); ?>

<?php endif; ?>

woocommerce.php
<?php
/**
 * Template Name: WooCommerce
 *
 * Please do not edit this file. This file is part of the CyberChimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */

get_header(); ?>

	<div id="primary" class="content-area container">
		<main id="main" class="site-main row" role="main">

			<div class="xs-col-12">
				<div class="page-content">
					<?php woocommerce_content(); ?>

						<?php get_template_part( 'content', 'page' ); ?>

						<?php
						// If comments are open or we have at least one comment, load up the comment template
						if( comments_open() || '0' != get_comments_number() ) {
							comments_template();
						}
						?>

				</div>
			</div>

		</main>
	</div><!-- #primary -->

<?php get_footer(); ?>

home-widgets.php
<?php
/**
 * Home Widgets Template
 *
 * Please do not edit this file. This file is part of the CyberChimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */
?>
<div id="home_widgets">
	<div class="container">

		<?php do_action( 'home_widgets_before' ); ?>

		<ul class="widget-area xs-col-12" role="complementary">

			<?php
			if ( is_active_sidebar( 'home-widgets' ) ) {
				// Get sidebar widgets & count footer widgets number
				$sidebars_widgets = wp_get_sidebars_widgets();
				$home_widget_count = count($sidebars_widgets['home-widgets']);
				switch ($home_widget_count) {
					case 2: $class = "xs-col-6"; break;
					case 3: $class = "m-col-4"; break;
					case 4: $class = "s-col-3"; break;
					default: $class = "s-col-12";
				}
			} ?>

			<?php if( !dynamic_sidebar( 'home-widgets' ) ) : ?>

				<li class="widget-container s-col-3">
					<h3 class="widget-title"><?php _e('Pages', 'altitude' ); ?></h3>
					<ul>
						<?php wp_list_pages('title_li=' ); ?>
					</ul>
				</li>

				<li class="widget-container s-col-3">
					<h3 class="widget-title"><?php _e( 'Archives', 'altitude' ); ?></h3>
					<ul>
						<?php wp_get_archives('type=monthly'); ?>
					</ul>
				</li>

				<li class="widget-container s-col-3">
					<h3 class="widget-title"><?php _e('Categories', 'altitude' ); ?></h3>
					<ul>
						<?php wp_list_categories('show_count=1&title_li='); ?>
					</ul>
				</li>

				<li class="widget-container s-col-3">
					<h3 class="widget-title"><?php _e('Subscribe', 'altitude' ); ?></h3>
					<ul>
						<li><a href="<?php bloginfo('rss2_url'); ?>"><?php _e('Entries (RSS)', 'altitude' ); ?></a></li>
						<li><a href="<?php bloginfo('comments_rss2_url'); ?>"><?php _e('Comments (RSS)', 'altitude' ); ?></a></li>
					</ul>
				</li>

			<?php endif; // end sidebar widget area ?>

		</ul>

		<script type="text/javascript">
			jQuery('#home_widgets .widget-area > li').addClass('<?php echo $class; ?>');
		</script>

		<?php do_action( 'home_widgets_after' ); ?>

	</div><!-- .container -->
</div><!-- #home_widgets -->

inc/customizer.php
<?php
/**
 * Customizer
 *
 * Please do not edit this file. This file is part of the CyberChimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */

/**
 * Add postMessage support for site title and description for the Theme Customizer.
 *
 * @param WP_Customize_Manager $wp_customize Theme Customizer object.
 */
function altitude_customize_register( $wp_customize ) {
	$wp_customize->get_setting( 'blogname' )->transport         = 'postMessage';
	$wp_customize->get_setting( 'blogdescription' )->transport  = 'postMessage';
	$wp_customize->get_setting( 'header_textcolor' )->transport = 'postMessage';

	// Theme Options
	$wp_customize->add_section( 'altitude_theme_options', array(
		'title'    => __( 'Theme Options', 'altitude' ),
		'priority' => 30
	) );

	// Home Page Button
	$wp_customize->add_setting( 'altitude_home_button_text', array(
		'default'   => __( 'View Themes & Pricing', 'altitude' ),
	) );

	$wp_customize->add_control( 'altitude_home_button_text', array(
		'label'    => __( 'Home Button Text', 'altitude' ),
		'section'  => 'altitude_theme_options',
		'type'     => 'text',
		'settings' => 'altitude_home_button_text'
	) );

	$wp_customize->add_setting( 'altitude_home_button_link', array(
		'default'   => '#',
	) );

	$wp_customize->add_control( 'altitude_home_button_link', array(
		'label'    => __( 'Home Button Link', 'altitude' ),
		'section'  => 'altitude_theme_options',
		'type'     => 'text',
		'settings' => 'altitude_home_button_link'
	) );

	// Home Page Title
	$wp_customize->add_setting( 'altitude_home_title', array(
		'default'   => __( 'Altitude Features', 'altitude' ),
	) );

	$wp_customize->add_control( 'altitude_home_title', array(
		'label'    => __( 'Home Page Title', 'altitude' ),
		'section'  => 'altitude_theme_options',
		'type'     => 'text',
		'settings' => 'altitude_home_title'
	) );

	$wp_customize->add_setting( 'altitude_home_sub_title', array(
		'default'   => __( 'Packed full of great features, here are just a few', 'altitude' ),
	) );

	$wp_customize->add_control( 'altitude_home_sub_title', array(
		'label'    => __( 'Home Page Subtitle', 'altitude' ),
		'section'  => 'altitude_theme_options',
		'type'     => 'text',
		'settings' => 'altitude_home_sub_title'
	) );

	// Home Page Button
	$wp_customize->add_setting( 'altitude_home_button_text', array(
		'default'   => __( 'View Themes & Pricing', 'altitude' ),
	) );

	$wp_customize->add_control( 'altitude_home_button_text', array(
		'label'    => __( 'Home Button Text', 'altitude' ),
		'section'  => 'altitude_theme_options',
		'type'     => 'text',
		'settings' => 'altitude_home_button_text'
	) );

	// Social Icons
	$wp_customize->add_section( 'altitude_social_options', array(
		'title'    => __( 'Social Options', 'altitude' ),
		'priority' => 35
	) );

	$wp_customize->add_setting( 'altitude_twitter', array(
		'default'   => '',
	) );

	$wp_customize->add_control( 'altitude_twitter', array(
		'label'    => __( 'Twitter URL', 'altitude' ),
		'section'  => 'altitude_social_options',
		'type'     => 'text',
		'settings' => 'altitude_twitter'
	) );

	$wp_customize->add_setting( 'altitude_facebook', array(
		'default'   => '',
	) );

	$wp_customize->add_control( 'altitude_facebook', array(
		'label'    => __( 'Facebook URL', 'altitude' ),
		'section'  => 'altitude_social_options',
		'type'     => 'text',
		'settings' => 'altitude_facebook'
	) );

	$wp_customize->add_setting( 'altitude_google_plus', array(
		'default'   => '',
	) );

	$wp_customize->add_control( 'altitude_google_plus', array(
		'label'    => __( 'Google Plus URL', 'altitude' ),
		'section'  => 'altitude_social_options',
		'type'     => 'text',
		'settings' => 'altitude_google_plus'
	) );

	// Sidebar Location
	$wp_customize->add_setting( 'altitude_sidebar', array(
		'default'   => 'right',
	) );

	$wp_customize->add_control( 'altitude_sidebar', array(
		'label'    => __( 'Sidebar Location', 'altitude' ),
		'section'  => 'altitude_theme_options',
		'type'     => 'select',
		'choices'  => array(
			'left'  => __( 'Left', 'altitude' ),
			'right' => __( 'Right', 'altitude' )
		),
		'settings' => 'altitude_sidebar'
	) );

	// Add Logo Image Support
	$wp_customize->add_section( 'altitude_logo_image_options', array(
		'title'    => __( 'Logo Image', 'altitude' ),
		'priority' => 45
	) );

	$wp_customize->add_setting( 'altitude_logo_image', array(
		'default'  => get_template_directory_uri() . '/images/default_logo.png',
	) );

	$wp_customize->add_control(
		new WP_Customize_Image_Control(
			$wp_customize,
			'altitude_logo_image',
			array(
				'label'      => __( 'Upload Logo Image (size 126x28 pixels)', 'altitude' ),
				'section'    => 'altitude_logo_image_options',
				'settings'   => 'altitude_logo_image',
			)
		)
	);

	// Home Header Image Options
	$wp_customize->add_section( 'altitude_home_header_image_options', array(
		'title'    => __( 'Home Header Image', 'altitude' ),
		'priority' => 50
	) );

	$wp_customize->add_setting( 'altitude_home_header_image', array(
		'default'  => get_template_directory_uri() . '/images/home_header_image.jpg',
	) );

	$wp_customize->add_control(
		new WP_Customize_Image_Control(
			$wp_customize,
			'altitude_home_header_image',
			array(
				'label'      => __( 'Upload Home Header Image (size 2000x1006 pixels)', 'altitude' ),
				'section'    => 'altitude_home_header_image_options',
				'settings'   => 'altitude_home_header_image'
			)
		)
	);

	/**
	 * This will create a footer section in customizer.
	 * It will create two text boxes for take value to both left and right section of footer.
	 */
	$wp_customize->add_section(
		'altitude_footer_section',
		array(
			'title'       => __( 'Footer Section', 'altitude' ),
			'description' => 'This is a settings section for footer.',
			'priority'    => 100,
		)
	);

	$wp_customize->add_setting(
		'altitude_copyright_textbox',
		array(
			'default' => '',
			'sanitize_callback' => 'altitude_text_sanitization'
		)
	);

	$wp_customize->add_control(
		'altitude_copyright_textbox',
		array(
			'label'   => __( 'Copyright text', 'altitude' ),
			'section' => 'altitude_footer_section',
			'settings' => 'altitude_copyright_textbox',
			'type'    => 'text'
		)
	);

}
add_action( 'customize_register', 'altitude_customize_register' );

/**
 * Text field sanitization
 *
 * @param $text
 *
 * @return string
 */
function altitude_text_sanitization( $text ) {
	return sanitize_text_field( $text );
}

/**
 * Binds JS handlers to make Theme Customizer preview reload changes asynchronously.
 */
function altitude_customize_preview_js() {
	wp_enqueue_script( 'altitude_customizer', get_template_directory_uri() . '/js/customizer.js', array( 'customize-preview' ), '20130508', true );
}
add_action( 'customize_preview_init', 'altitude_customize_preview_js' );


inc/custom-header.php
<?php
/**
 * Custom header
 *
 * Please do not edit this file. This file is part of the CyberChimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */

/**
 * Setup the WordPress core custom header feature.
 *
 * @uses altitude_header_style()
 * @uses altitude_admin_header_style()
 * @uses altitude_admin_header_image()
 *
 * @package Altitude
 */
function altitude_custom_header_setup() {
	add_theme_support( 'custom-header', apply_filters( 'altitude_custom_header_args', array(
		'default-image'          => get_template_directory_uri() . '/images/default_header_image.jpg',
		'default-text-color'     => '000000',
		'width'                  => 2000,
		'height'                 => 506,
		'flex-height'            => true,
		'wp-head-callback'       => 'altitude_header_style',
		'admin-head-callback'    => 'altitude_admin_header_style',
		'admin-preview-callback' => 'altitude_admin_header_image',
	) ) );
}
add_action( 'after_setup_theme', 'altitude_custom_header_setup' );

if ( ! function_exists( 'altitude_header_style' ) ) :
	/**
	 * Styles the header image and text displayed on the blog
	 *
	 * @see altitude_custom_header_setup().
	 */
	function altitude_header_style() {
		$header_image      = get_header_image();
		$home_header_image = get_theme_mod( 'altitude_home_header_image', get_template_directory_uri() . '/images/home_header_image.jpg' );
		$header_text_color = get_header_textcolor();

		// If no custom options for text are set, let's bail
		// get_header_textcolor() options: HEADER_TEXTCOLOR is default, hide text (returns 'blank') or any hex value
		if ( HEADER_TEXTCOLOR != $header_text_color ) :

			// If we get this far, we have custom styles. Let's do this.
			?>
			<style type="text/css">
				<?php
					// Has the text been hidden?
					if ( 'blank' == $header_text_color ) :
				?>
				#site_branding_container .site-title,
				#site_branding_container .site-description {
					visibility: hidden;
				}

				<?php
					// If the user has set a custom color for the text use that
					else :
				?>
				#site_branding_container .site-title a,
				#site_branding_container .site-description {
					color: #<?php echo $header_text_color; ?>;
				}

				<?php endif; ?>
			</style>

		<?php
		endif;
		?>

		<style type="text/css">
			#masthead-inner {
				background-size: 100% 100%;
				background-position: 50% 50%;
				background-repeat: no-repeat;
			}
		</style>

		<?php
		$classes = get_body_class();
		if ( in_array('home',$classes) ) : ?>

			<style type="text/css">
				#masthead-inner {
					background-image: url( <?php echo $home_header_image ?> );
				}
				#masthead:before {
					padding-top: 50%;
				}
			</style>

		<?php else : ?>

			<?php if ( HEADER_IMAGE != $header_image && $header_image != false ) : ?>

				<style type="text/css">
					#masthead-inner {
						background-image: url( <?php echo $header_image ?> );
					}
				</style>

			<?php else : ?>

				<style type="text/css">
					#masthead-inner {
						background-image: url( <?php echo HEADER_IMAGE ?> );
					}
				</style>

			<?php endif; ?>

		<?php endif;

		return;
	}
endif; // altitude_header_style

if ( ! function_exists( 'altitude_admin_header_style' ) ) :
	/**
	 * Styles the header image displayed on the Appearance > Header admin panel.
	 *
	 * @see altitude_custom_header_setup().
	 */
	function altitude_admin_header_style() {
	?>
		<style type="text/css">
			.appearance_page_custom-header #headimg {
				position: relative;
				width: 100%;
				border: none;
				background-color: #302e32;
			}

			.appearance_page_custom-header #headimg:before {
				content: "";
				display: block;
				padding-top: 25%;
			}

			.appearance_page_custom-header #headimg .header-img-bg {
				position: absolute;
				top: 0;
				left: 0;
				bottom: 0;
				right: 0;
				max-width: 100%;
				z-index: 1;
			}

			.appearance_page_custom-header .headtxt {
				position: absolute;
				top: 53%;
				left: 0;
				right: 0;
				z-index: 2;
				text-align: center;
				font-family: 'Source Sans Pro', serif; font-size: 16px;
				color: #FFFFFF;
			}

			.appearance_page_custom-header .headtxt h1 {
				font-size: 3.5vw;
				font-weight: 200;
				line-height: 1;
				margin: 0;
			}

			.appearance_page_custom-header .headtxt h2 {
				font-size: 1.6vw;
				font-weight: 100;
				padding: 0;
				color: #FFFFFF;
				line-height: 1;
			}

			.appearance_page_custom-header #main_navigation_container {
				position: absolute;
				top: 0;
				left: 0;
				right: 0;
				z-index: 3;
			}

			.appearance_page_custom-header #navigation-inner {
				position: relative;
				width: 70%;
				margin: 0 auto;
				background: #000000;
				padding: 35px;
				overflow: hidden;
			}

			.appearance_page_custom-header #site_logo,
			.appearance_page_custom-header #social_container {
				float: left;
				width: 14%;
			}

			.appearance_page_custom-header #site_logo img {
				max-width: 100%;
			}

			.appearance_page_custom-header #site_navigation {
				float: left;
				width: 66%;
				margin: 0 3%;
			}

			.appearance_page_custom-header #site_navigation .menu {
				padding-top: 10px;
			}

			.appearance_page_custom-header #site_navigation .menu ul {
				margin: 0;
				padding: 0;
				list-style-type: none;
			}

			.appearance_page_custom-header #site_navigation .menu ul li {
				margin-bottom: 0;
				position: relative;
				float: left;
				display: inline-block;
				margin-left: 12px;
			}

			.appearance_page_custom-header #site_navigation .menu ul li:first-child {
				margin-left: 0;
			}

			.appearance_page_custom-header #site_navigation .menu ul li a {
				color: #c7c7c7;
				font-weight: 300;
				text-decoration: none;
				font-size: 0.75vw;
			}

			.appearance_page_custom-header #site_navigation .menu ul li a:hover,
			.appearance_page_custom-header #site_navigation .menu ul li a:focus {
				color: #FFFFFF;
			}

			.appearance_page_custom-header #social_container .social-icons {
				float: right;
				list-style-type: none;
				margin: 0;
				padding: 0;
			}

			.appearance_page_custom-header #social_container .social-icons li {
				float: left;
				margin-left: 25px;
				margin-bottom: 0;
			}

			.appearance_page_custom-header #social_container .social-icons li:first-child {
				margin-left: 0;
			}

			.appearance_page_custom-header #social_container .social-icons li a {
				font-size: 1.3vw;
				color: #565656;
			}

			.appearance_page_custom-header #social_container .social-icons li a:hover {
				color: #FFFFFF;
			}

			@media (max-width: 1500px) {
				.appearance_page_custom-header #navigation-inner {
					width: 90%;
				}

				.appearance_page_custom-header #site_navigation .menu ul li a {
					font-size: 0.9vw;
				}

				.appearance_page_custom-header #social_container .social-icons li a {
					font-size: 1.6vw;
				}
			}
		</style>
	<?php
	}
endif; // altitude_admin_header_style

if ( ! function_exists( 'altitude_admin_header_image' ) ) :
	/**
	 * Custom header image markup displayed on the Appearance > Header admin panel.
	 *
	 * @see altitude_custom_header_setup().
	 */
	function altitude_admin_header_image() {
		$style        = sprintf( ' style="color:#%s;"', get_header_textcolor() );
		$header_image = get_header_image();
		?>
		<div id="headimg">

			<div id="main_navigation_container" class="container">
				<div id="navigation-inner">

					<div id="site_logo">
						<a href="<?php echo esc_url( home_url( '/' ) ); ?>" title='<?php echo esc_attr( get_bloginfo( 'name', 'display' ) ); ?>' rel="home" onclick="return false;">
							<img src='<?php echo esc_url( get_theme_mod( 'altitude_logo_image', get_template_directory_uri() . '/images/default_logo.png' ) ); ?>' alt='<?php echo esc_attr( get_bloginfo( 'name', 'display' ) ); ?>'>
						</a>
					</div><!-- #site-logo -->

					<nav id="site_navigation" role="navigation">
						<?php wp_nav_menu( array( 'theme_location' => 'primary' ) ); ?>
					</nav><!-- #site-navigation -->

					<div id="social_container">
						<ul class="social-icons">
							<?php if ( '' != get_theme_mod( 'jetpack-twitter' ) ): ?>
								<li class="altitude-twitter">
									<a href="<?php echo esc_url( get_theme_mod( 'jetpack-twitter' ) ); ?>" title="<?php _e( 'Twitter', 'altitude' ); ?>">
										<i class="fa fa-twitter"></i>
									</a>
								</li>
							<?php endif; ?>
							<?php if ( '' != get_theme_mod( 'jetpack-facebook' ) ): ?>
								<li class="altitude-facebook">
									<a href="<?php echo esc_url( get_theme_mod( 'jetpack-facebook' ) ); ?>" title="<?php _e( 'Facebook', 'altitude' ); ?>">
										<i class="fa fa-facebook"></i>
									</a>
								</li>
							<?php endif; ?>
							<?php if ( '' != get_theme_mod( 'jetpack-google_plus' ) ): ?>
								<li class="altitude-google-plus">
									<a href="<?php echo esc_url( get_theme_mod( 'jetpack-google_plus' ) ); ?>" title="<?php _e( 'Google Plus', 'altitude' ); ?>">
										<i class="fa fa-google-plus"></i>
									</a>
								</li>
							<?php endif; ?>
						</ul><!-- .social-icons -->
					</div>

				</div>
			</div><!-- navigation container -->

			<div class="headtxt">
				<h1 class="site-title"><?php bloginfo( 'name' ); ?></h1>
				<h2 class="site-description"><?php bloginfo( 'description' ); ?></h2>
			</div>

			<?php if( !empty( $header_image ) ) : ?>
				<img class="header-img-bg" src="<?php echo esc_url( $header_image ); ?>" alt="">
			<?php endif; ?>
		</div>
	<?php
	}
endif; // altitude_admin_header_image

functions.php
<?php
/**
 * Title: Function
 *
 * Description: Defines theme specific functions including actions and filters.
 *
 * Please do not edit this file. This file is part of the CyberChimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v2.0 (or later)
 * @link     http://www.cyberchimps.com/
 */

$template_directory = get_template_directory();

// Notify user of theme update on "Updates" page in Dashboard.
//require_once( $template_directory . '/inc/wp-updates-theme.php');
//new WPUpdatesThemeUpdater_483( 'http://wp-updates.com/api/2/theme', basename( $template_directory ) );

/**
 * Set the content width based on the theme's design and stylesheet.
 */
if ( ! isset( $content_width ) ) {
	$content_width = 640; /* pixels */
}

if ( ! function_exists( 'altitude_setup' ) ) :
/**
 * Sets up theme defaults and registers support for various WordPress features.
 *
 * Note that this function is hooked into the after_setup_theme hook, which
 * runs before the init hook. The init hook is too late for some features, such
 * as indicating support for post thumbnails.
 */
function altitude_setup() {

	/**
	 * Make theme available for translation.
	 * Translations can be filed in the /languages/ directory.
	 * If you're building a theme based on Altitude, use a find and replace
	 * to change 'altitude' to the name of your theme in all the template files
	 */
	load_theme_textdomain( 'altitude', get_template_directory() . '/languages' );

	/**
	 * Add default posts and comments RSS feed links to head
	 */
	add_theme_support( 'automatic-feed-links' );

	/**
	 * Enable support for Post Thumbnails on posts and pages
	 *
	 * @link http://codex.wordpress.org/Function_Reference/add_theme_support#Post_Thumbnails
	 */
	add_theme_support( 'post-thumbnails' );

	/**
	 * This theme uses wp_nav_menu() in one location.
	 */
	register_nav_menus( array(
		'primary' => __( 'Primary Menu', 'altitude' ),
		'footer'  => __( 'Footer Menu', 'altitude' )
	) );

	/**
	 * Enable support for Post Formats
	 */
	add_theme_support( 'post-formats', array( 'aside', 'image', 'video', 'quote', 'link' ) );

	/**
	 * Add new thumbnails size
	 */
	add_image_size( 'altitude-home', '706', '240', true );
	add_image_size( 'altitude-blog', '1170', '283', true );
	add_image_size( 'altitude-post', '848', '205', true );

	/**
	 * Add stylesheet to post/page edit
	 */
	add_editor_style( 'layouts/editor-style.css' );

	/**
	 * Enable support for HTML5 markup.
	 */
	add_theme_support( 'html5', array(
		'comment-list',
		'search-form',
		'comment-form',
		'gallery',
		'caption',
	) );
}
endif; // altitude_setup
add_action( 'after_setup_theme', 'altitude_setup' );

/**
 * Register widgetized area and update sidebar with default widgets
 */
function altitude_widgets_init() {
	register_sidebar( array(
		'name'          => __( 'Sidebar', 'altitude' ),
		'id'            => 'sidebar-1',
		'before_widget' => '<aside id="%1$s" class="widget %2$s">',
		'after_widget'  => '</aside>',
		'before_title'  => '<h3 class="widget-title">',
		'after_title'   => '</h3>',
	) );

	register_sidebar( array(
		'name'          => __( 'Home Widgets', 'altitude' ),
		'id'            => 'home-widgets',
		'description'   => __( 'Home Page Bottom Widgets Area', 'altitude' ),
		'before_widget' => '<li id="%1$s" class="widget %2$s">',
		'after_widget'  => '</li>',
		'before_title'  => '<h3 class="widget-title">',
		'after_title'   => '</h3>',
	) );
}
add_action( 'widgets_init', 'altitude_widgets_init' );

/**
 * Enqueue scripts and styles.
 */
function altitude_scripts() {

	$template_directory_uri = get_template_directory_uri();

	// Check if it exists in child theme.
	if( file_exists( get_stylesheet_directory() . '/layouts/core.css') ) {
		wp_enqueue_style( 'altitude-core', get_stylesheet_directory_uri() . '/layouts/core.css', array(), '20131022' );
	}
	else {
		wp_enqueue_style( 'altitude-core', $template_directory_uri . '/layouts/core.css', array(), '20131022' );
	}

	wp_enqueue_style( 'altitude-style', get_stylesheet_uri() );

	wp_enqueue_script( 'altitude-skip-link-focus-fix', $template_directory_uri . '/js/skip-link-focus-fix.js', array(), '20130115', true );

	if( is_singular() && comments_open() && get_option( 'thread_comments' ) ) {
		wp_enqueue_script( 'comment-reply' );
	}

	if( is_singular() && wp_attachment_is_image() ) {
		wp_enqueue_script( 'altitude-keyboard-image-navigation', $template_directory_uri . '/js/keyboard-image-navigation.js', array( 'jquery' ), '20120202' );
	}

	wp_enqueue_script( 'altitude-stellar', $template_directory_uri . '/js/jquery.stellar.min.js', array( 'jquery' ), '20130825', true );

	wp_enqueue_script( 'altitude-isotope', get_template_directory_uri() . '/js/isotope.pkgd.min.js', array(), '20140630', true );

	wp_enqueue_script( 'altitude-theme', $template_directory_uri . '/js/theme-js.js', array( 'jquery', 'altitude-stellar' ), '20130825', true );

	// Add Sanchez google font, used in the main stylesheet.
	wp_enqueue_style( 'altitude-sanchez' );

	wp_enqueue_script( 'altitude-navigation', get_template_directory_uri() . '/js/navigation.js', array(), '20120206', true );
}
add_action( 'wp_enqueue_scripts', 'altitude_scripts' );

add_theme_support( 'woocommerce' );
/**
 * Customizer additions.
 */
require $template_directory . '/inc/customizer.php';

/**
 * Implement the Custom Header feature.
 */
require $template_directory . '/inc/custom-header.php';

/**
 * Custom template tags for this theme.
 */
require $template_directory . '/inc/template-tags.php';

/**
 * Custom functions that act independently of the theme templates.
 */
require $template_directory . '/inc/extras.php';

/**
 * Load Jetpack compatibility file.
 */
require $template_directory . '/inc/jetpack.php';

/**
 * Admin code
 */
require $template_directory . '/inc/admin.php';

/**
 * Used to display the contact details saved in theme_mods
 * change in customizer
 *
 * @param $type string phone, address or email
 *
 * @echos string
 */
function altitude_contact_details( $type ) {

	switch( $type ) {
		case 'phone':
			$contact = get_theme_mod( 'altitude_phone_number' );
			$render  = '<div class="contact-item">' . esc_html( $contact ) . '</div>';
			break;
		case 'address':
			$contact = get_theme_mod( 'altitude_address' );
			$render  = '<div class="altitude-item">' . esc_html( $contact ) . '</div>';
			break;
		case 'email':
			$email   = get_theme_mod( 'altitude_email' );
			$contact = '<a href="mailto:' . antispambot( sanitize_email( $email ) ) . '" title="' . __( 'email address', 'altitude' ) . '">' . esc_html( $email ) . '</a>';
			$render  = '<div class="altitude-item">' . $contact . '</div>';
			break;
		default:
			$render = '';
	}

	echo $render;
}

/**
 * function decides where to put the sidebar. Left or right of content
 *
 * @param $loc string - left or right
 */
function get_altitude_sidebar( $loc ) {
	$location = get_theme_mod( 'altitude_sidebar' );
	$location = $location ? $location : 'right';

	if( $loc == $location ) {
		get_sidebar();
	}
}

/**
 * Register Styles
 */
function altitude_register_styles() {
	wp_register_style( 'altitude-sourcesanspro', altitude_google_font_url(), array(), null );
	wp_register_style( 'altitude-fontawesome', '//netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css' );
}
add_action( 'init', 'altitude_register_styles' );

/**
 * Enqueue Styles
 */
function altitude_load_styles( $hook_suffix ) {
	wp_enqueue_style( 'altitude-sourcesanspro' );
	wp_enqueue_style( 'altitude-fontawesome' );

	// Add Genericons font, used in the main stylesheet.
	wp_enqueue_style( 'genericons', get_template_directory_uri() . '/genericons/genericons.css', array(), '3.0.2' );
}
add_action( 'wp_print_styles', 'altitude_load_styles' );


/**
 * Enqueue Admin Scripts for custom headers
 */
function altitude_admin_scripts( $hook_suffix ) {
	if ( 'appearance_page_custom-header' != $hook_suffix )
		return;

	wp_enqueue_style( 'altitude-sourcesanspro' );
	wp_enqueue_style( 'altitude-fontawesome' );
}
add_action( 'admin_enqueue_scripts', 'altitude_admin_scripts' );

/**
 * Create the font urls
 */
function altitude_google_font_url() {
	$font_url = '';
	/*
	 * Translators: If there are characters in your language that are not supported
	 * by Source Sans Pro or font awesome, translate this to 'off'. Do not translate into your own language.
	 */
	if ( 'off' !== _x( 'on', 'source sans pro font: on or off', 'altitude' ) ) {
		$font_url = add_query_arg( 'family', 'Source+Sans+Pro:200,300,400,600,700,900', "//fonts.googleapis.com/css" );
	}

	return $font_url;
}


/**
 * Customize excerpt more
 */
function altitude_excerpt_more( $more ) {
	return '... <a class="read-more" href="'. get_permalink( get_the_ID() ) . '">' . __('Continue Reading', 'altitude') . '</a>';
}
add_filter( 'excerpt_more', 'altitude_excerpt_more' );